code/modules/clothing/head/hat.dm

```diff
@@ -53,11 +53,11 @@
 
 /obj/item/clothing/head/bio_hood/plague
```

Comment:
I'm looking at `/obj/item/clothing/head/bio_hood` and actually I am not sure this item gets much out of being a child of this type at all.

What it's getting from here is:
- FOV
- flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH | PEPPERPROOF
- resistance_flags = ACID_PROOF

As it is, it does not look like you have modified `flags_cover` so wearing this hat will cover your eyes and mouth and make you resistant to pepper spray, and also the hat will be resistant to acid.
I don't care either wya about whether this hat is acidproof, but for the other you will either want to:

- add `flags_cover = NONE`
- change the path to `/obj/item/clothing/head/plague`

The latter is probably the better solution but is more complicated because it will have side-effects, namely everywhere currently referencing this item including maps will need to be updated and you would also probably need to write a script which can be run to replace the path in every map on our repo and any downstream repo (this isn't hard but you would need to look up how to do it).

Sorry for uh, bringing this up on what you assumed would be a two minute PR 😅

Reply:
I am learning which is the goal!